.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,15 +1,22 @@
 ---
 name: Bug Report
 about: Report a bug/crash
-title: ''
+title: '[Bug] A brief description about problem you found'
```

Comment:
```suggestion
title: ''
```
The only thing of value this adds is an encouragement to use [x] psudo-tags. Those are not necessary or wanted. We have real tags, and we can decide if what is being described is a bug (it's usually obvious).